src/main/java/rife/test/MockConversation.java

```diff
@@ -85,7 +85,7 @@ public MockResponse doRequest(String url)
      * Perform a request for a particular URL and request configuration.
      * <p>The request can either be complete with the scheme and hostname, or
      * an absolute path. These two URLs are thus considered the same:
-     * <pre>https://localhost/some/url?name1=value1&amp;name2=value2</pre>
+     * <pre><a href="https://localhost/some/url?name1=value1&amp;name2=value2">...</a></pre>
```

Comment:
What's the point of the HTML a tag here?

Reply:
None.